### PR TITLE
Enabled login with email only.

### PR DESCRIPTION
**Pull request by arshadkhan53**
Enabled login with email only.

### CODE REVIEW

File: modules/ezcontent_user/ezcontent_user.info.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: EZContent USER
```
Comment (vaibhavjain-in) [inline modules/ezcontent_user/ezcontent_user.info.yml]: Can the name be `EZContent User Feature`

File: modules/ezcontent_user/ezcontent_user.info.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+type: module
+description: 'Alter user forms and manage user entity.'
```
Comment (vaibhavjain-in) [inline modules/ezcontent_user/ezcontent_user.info.yml]: Let the description be `Provides user based features.`

File: modules/ezcontent_user/ezcontent_user.info.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+package: EZContent USER
```
Comment (vaibhavjain-in) [inline modules/ezcontent_user/ezcontent_user.info.yml]: Let the package name be `EZContent User`

**Review by vaibhavjain-in — CHANGES_REQUESTED** [review]
@arshadkhan53 Please see some minor issues. Please cater to the feedback.

**Review by vaibhavjain-in — APPROVED** [review]
Looks good.

File: composer.json [hunk 4]
```diff
@@ -46,6 +46,7 @@
     "drupal/length_indicator": "^1.0",
     "drupal/libraries": "3.0-alpha1",
     "drupal/linkit": "~5.0",
+    "drupal/login_onlyemail": "^1.5",
```
Comment (ddfiberglass) [inline composer.json]: I have used docker images for EzContent installation. What is the default email and password for log in?

The installation steps is exact as (https://hub.docker.com/r/srijanlabs/ezcontent) below:

Create a network
docker network create ez-nw

Create containers
docker run --rm --name mysql -e MYSQL_ROOT_PASSWORD=drupal -e MYSQL_USER=drupal -e MYSQL_PASSWORD=drupal -e MYSQL_DATABASE=drupal --network ez-nw -p 3306:3306 -d mysql:5.7
docker run --rm --name ezcontent_drupal --network ez-nw -e DB_HOST=mysql -e DB_USER=drupal -e DB_PASSWORD=drupal -e DB_PORT=3306 -e DB_NAME=drupal -p 8062:8080 -d srijanlabs/ezcontent:8.x
Run following command to install the Ezcontent Stack
docker exec -it ezcontent_drupal sh -c "php -d memory_limit=-1 ./vendor/bin/drush si ezcontent --db-url='mysql://drupal:drupal@mysql/drupal' --site-name='EZ Content' --account-name='srijan' --account-pass='admin12345' -y && php -d memory_limit=-1 ./vendor/bin/drush en ezcontent_demo -y && php -d memory_limit=-1 ./vendor/bin/drush en ezcontent_api -y && php -d memory_limit=-1 ./vendor/bin/drush en ezcontent_api -y"

**Pull request merged**